requirements-test.txt

```diff
@@ -0,0 +1,2 @@
+pytest
+flake8
```

Comment:
Curious why you stuff these guys in requirements files vs raw setuptools (`install_requires` and `tests_require` sections)?